src/profilers/profile-serializer.ts

```diff
@@ -87,9 +83,7 @@ export function serializeTimeProfile(
           });
           samples.push(sample);
         }
-        for (let child of node.children) {
-          entries.push({node: child, stack: stack.slice(0)});
-        }
+        node.children.map((n) => entries.push({node: n, stack: stack.slice(0)}));
```

Comment:
Same comment as above regarding `map`, but you can also take advantage of the spread params operator (that `f(...[a,b,c])` and `f(a,b,c)` are equivalent) here: `entries.push(...node.children.map(n=>({node, stack})))`